clients/src/main/java/org/apache/kafka/common/requests/FetchResponse.java

```diff
@@ -58,8 +58,8 @@
  *
  * - {@link Errors#OFFSET_OUT_OF_RANGE} If the fetch offset is out of range for a requested partition
  * - {@link Errors#TOPIC_AUTHORIZATION_FAILED} If the user does not have READ access to a requested topic
- * - {@link Errors#REPLICA_NOT_AVAILABLE} If the request is received by a broker which is not a replica
- * - {@link Errors#NOT_LEADER_FOR_PARTITION} If the broker is not a leader and either the provided leader epoch
+ * - {@link Errors#REPLICA_NOT_AVAILABLE} If the request is received by a broker with version 2.4 to 2.6 which is not a replica
```

Comment:
Is the note about the range from 2.4 to 2.6 correct? I think this error has always been possible in the case of a reassignment.

Reply:
Updated to say < 2.6.